File: level1/javascript/제일 작은 수 제거하기 12935.js
function solution(arr) {
    if (arr.length == 1)
        return [-1];

    arr.splice(arr.indexOf(Math.min(...arr)), 1);
    return arr;
}

// 테스트 코드
console.log(solution([1, 2, 3, 4])); // [4,3,2]
console.log(solution([4, 3, 2, 1])); // [4,3,2]
console.log(solution([10])); // [-1]

File: level1/kotlin/제일 작은 수 제거하기 12935.kt
class Solution {
    fun solution(arr: IntArray): IntArray {
        if (arr.size == 1) return intArrayOf(-1)

        val answer = arr.toMutableList()
        answer.remove(arr.minOrNull() ?: return intArrayOf(-1))
        return answer.toIntArray()
    }
}